### PR TITLE
Added Legal team email to the Contact Page

### PR DESCRIPTION
**Pull request by VenierGiacomo**
As requested by Nico, I have added the legal team email on the `Contact` page

image

### CODE REVIEW

File: src/components/ContactUs/Emails.tsx [hunk 1]
```diff
@@ -38,5 +38,11 @@ export const Emails = () => (
       </h6>
       <a href="mailto:contact@ledgy.com">contact@ledgy.com</a>
     </div>
+    <div className="mb-4">
```
Comment (almasen) [inline src/components/ContactUs/Emails.tsx:41]: This styling should go on the previous div, see spacing in screenshot:
image

**Review by almasen — COMMENTED** [review]
Please see above comment

File: src/components/ContactUs/Emails.tsx (hunk 1 above)
Comment (VenierGiacomo) [reply src/components/ContactUs/Emails.tsx]: Thanks :)

**Review by ovcOS — APPROVED** [review]
🧑‍⚖️

**Pull request merged**